### PR TITLE
Add navbar

### PR DESCRIPTION
**Pull request by guilleSequeiraWolox**
## Summary

Add home and navbar with logout.

## Screenshots

add home

## Trello Card

https://trello.com/c/nTt0x7xq/67-session-management-logout

### CODE REVIEW

File: react/wbooks/src/app/screens/Home/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import { Link } from 'react-router-dom';
+import { t } from 'i18next';
+
+import logoWolox from '../../assets/LogoWolox.png';
+
+import styles from './styles.module.scss';
+
+const clearTokens = () => {
+  localStorage.setItem('token_id', '');
+};
```
Comment (LucasZibell) [inline react/wbooks/src/app/screens/Home/index.js]: `const clearTokens = () => localStorage.setItem('token_id', '');`

Comment (LucasZibell) [reply react/wbooks/src/app/screens/Home/index.js]: I think there is a `localStorage.delete` or something like that 🤔

File: react/wbooks/src/app/screens/SignUp/index.js [hunk 2]
```diff
@@ -14,26 +14,28 @@ import actionCreators from './reducer/actions';
...
+  const onSubmitSignUp = useCallback(
+    values => {
+      createUser(values).then(response => {
+        if (response.ok) {
+          setToken(response.data.access_token || '');
+          dispatch(actionCreators.createUser(response));
+          setTimeout(() => {
+            alert(JSON.stringify(t('SIGNUP:successRegistrer'), null, 2));
+            history.push('/');
+          }, 1000);
```
Comment (LucasZibell) [inline react/wbooks/src/app/screens/SignUp/index.js]: Why are you setting a timout?

File: react/wbooks/src/app/screens/Login/index.js [hunk 3]
```diff
@@ -14,24 +14,25 @@ import actionCreators from './reducer/actions';
...
+      login(values).then(response => {
+        if (response.ok) {
+          setToken(response.data.access_token || '');
+          dispatch(actionCreators.login(response));
+          history.push('/home');
+        } else {
+          setTimeout(() => {
+            alert(JSON.stringify(t('LOGIN:error'), null, 2));
+          }, 1000);
+        }
```
Comment (LucasZibell) [inline react/wbooks/src/app/screens/Login/index.js:29]: timeout? 🤔

**Review by LucasZibell — CHANGES_REQUESTED** [review]

File: react/wbooks/src/app/screens/Home/index.js (hunk 1 above)
Comment (mariadelacamara) [reply react/wbooks/src/app/screens/Home/index.js]: localStorage.clear I think

**Review by LucasZibell — APPROVED** [review]

**Pull request merged**